deploy/sre-prometheus/aws/sts-privatelink/config.yaml

```diff
@@ -4,5 +4,8 @@ selectorSyncSet:
   - key: api.openshift.com/sts
     operator: NotIn
     values: ["true"]
+  - key: api.openshift.com/private-link
```

Comment:
I believe this needs an OR as part of the match

Reply:
Will retest the combinations of sts/private-link =true/false/not set. 
If the above needs and OR, do you think these conditions need to change, too : https://github.com/openshift/managed-cluster-config/blob/master/deploy/cloud-ingress-operator-configuration/sshd/config.yaml

Reply:
Actually after looking at this for now, I think it will work as expected.

Reply:
for completeness: the test matrix for the conditions:

![image](https://user-images.githubusercontent.com/5159809/123070919-79a19880-d414-11eb-97fe-f340841be1ff.png)